documentation/graphing-overview/index.html.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Flow.js and Data Visualization: Graphing with Contour"
+layout: "flow"
+isPage: true
+---
+
+##Flow.js and Data Visualization: Graphing with Contour
+
+
+Flow.js makes it easy to show data from your model variables in [text](../generated/dom/attributes/binds/default-bind-attr/) or [lists](../generated/dom/attributes/foreach/default-foreach-attr/). However, often you want more than just textual output &mdash; you want to visualize the results of your simulation!
+
+One way to do this is to use <a href="http://forio.com/contour/" target="_blank">Contour</a>, an open source JavaScript library developed by Forio for interactive data visualization, based on <a href="http://d3js.org" target="_blank">d3.js</a>. Most of the [example projects](../../project_admin/#personal) in Epicenter use Contour for their graphs.
+
+**To add a visualization to a page in your project:**
+
+1. Include the Contour.js library and its dependencies on the page.
+2. Create a new Contour chart on the page.
+3. Use the Flow.js [Variables Channel](../generated/channels/variables-channel/) to subscribe to the model variable you want to graph.
+4. In the callback function for your subscription, update and render the Contour chart.
+
+Optional extensions include:
+
+* Graphing multiple variables updated by the model.
+* Graphing multiple variables entered by the user.
+
+Let's look at each step in more detail. You can also jump straight to the [complete example](#example).
+
+
+**Step 1: Include the Contour.js library on the page.**
+
+Add `contour.js` and `contour.css` to the page of your project where the visualization appears. Additionally, include `d3.js` and `lodash.js`, which are required by Contour.
+
+	<html>
+		<head>
+			<link rel="stylesheet" href="https://forio.com/tools/contour/contour.min.css">
+			<script src="yourPath/d3.min.js"></script>
```

Comment:
for epi-js and flow, i put them in the complete example at the bottom, but think it's more clear to leave out here. will see what others think in demo mtg too.